# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/viewsource/nsViewSourceChannel.cpp
# Commit: f2df73fda468
# Full Hash: f2df73fda46859150605f33bcdaeb49ab7037720
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-10-19 21:40:34
# Regressor Bug: 1588899
# File Overlap Count: 1
# Description:
#   Bug 1588899 - P1. Move classification flags related method to nsIClassifiedChannel. r=Ehsan,baku
#   
#   This is where it should have been in the first place. Those attributes belong there.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49577
# ==============================================================================

diff -r b98968eb71d3 -r f2df73fda468 netwerk/protocol/viewsource/nsViewSourceChannel.cpp
--- a/netwerk/protocol/viewsource/nsViewSourceChannel.cpp	Thu Oct 17 15:24:31 2019 +0000
+++ b/netwerk/protocol/viewsource/nsViewSourceChannel.cpp	Sat Oct 19 04:30:24 2019 +0000
@@ -696,42 +696,6 @@
 }
 
 NS_IMETHODIMP
-nsViewSourceChannel::IsTrackingResource(bool* aIsTrackingResource) {
-  return !mHttpChannel ? NS_ERROR_NULL_POINTER
-                       : mHttpChannel->IsTrackingResource(aIsTrackingResource);
-}
-
-NS_IMETHODIMP
-nsViewSourceChannel::IsThirdPartyTrackingResource(bool* aIsTrackingResource) {
-  return !mHttpChannel
-             ? NS_ERROR_NULL_POINTER
-             : mHttpChannel->IsThirdPartyTrackingResource(aIsTrackingResource);
-}
-
-NS_IMETHODIMP
-nsViewSourceChannel::GetClassificationFlags(uint32_t* aClassificationFlags) {
-  return !mHttpChannel
-             ? NS_ERROR_NULL_POINTER
-             : mHttpChannel->GetClassificationFlags(aClassificationFlags);
-}
-
-NS_IMETHODIMP
-nsViewSourceChannel::GetFirstPartyClassificationFlags(
-    uint32_t* aClassificationFlags) {
-  return !mHttpChannel ? NS_ERROR_NULL_POINTER
-                       : mHttpChannel->GetFirstPartyClassificationFlags(
-                             aClassificationFlags);
-}
-
-NS_IMETHODIMP
-nsViewSourceChannel::GetThirdPartyClassificationFlags(
-    uint32_t* aClassificationFlags) {
-  return !mHttpChannel ? NS_ERROR_NULL_POINTER
-                       : mHttpChannel->GetThirdPartyClassificationFlags(
-                             aClassificationFlags);
-}
-
-NS_IMETHODIMP
 nsViewSourceChannel::GetFlashPluginState(
     nsIHttpChannel::FlashPluginState* aResult) {
   return !mHttpChannel ? NS_ERROR_NULL_POINTER